src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -226,6 +228,16 @@ UserImportResult importUsers(UserImportRequest request) throws FirebaseAuthExcep
     return new UserImportResult(request.getUsersCount(), response);
   }
 
+  Tenant getTenantById(String tenantId) throws FirebaseAuthException {
+    GenericUrl url = new GenericUrl(tenantMgtBaseUrl + "/tenants:get");
+    url.put("name", tenantId);
```

Comment:
This doesn't quite look right. The URL path should be something like `v2/projects/{projectsId}/tenants/{tenantsId}`. That's what we use in other SDKs, and also what's listed in the auto-generated discovery doc.

Reply:
Thanks! Fixed.